roles/network_plugin/weave/templates/weave-net.yml.j2

```diff
@@ -109,16 +109,16 @@ items:
         name: weave-net
       namespace: kube-system
     spec:
-      minReadySeconds: 5
+      # Wait 5 seconds to let pod connect before rolling next pod
```

Comment:
The original manifest doesn't contain this comment, and the order of `minReadySecounds` was the same as the old one.